src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/model/CaseRecordFactory.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model;
+
+import com.google.common.collect.ImmutableMap;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.caserecord.ExceptionRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.caserecord.SupplementaryEvidenceRecord;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.EnumMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.CaseRecord.Construct;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.model.CaseRecord.Record;
+
+public final class CaseRecordFactory {
```

Comment:
You will either have a exception record or update the case why do we need a factory ?